token-lending/program/src/processor.rs

```diff
@@ -427,7 +427,12 @@ fn process_refresh_reserve(program_id: &Pubkey, accounts: &[AccountInfo]) -> Pro
     let account_info_iter = &mut accounts.iter().peekable();
```

Comment:
also we should test:
- calling refresh reserve with clock and switchboard
- clock and no switchboard
- no clock and no switchboard